arclightning_backend/src/router.rs

```diff
@@ -143,4 +143,25 @@ impl Router {
             _ => self.invalid_endpoint(),
         }
     }
+
+    fn list_files(dir: PathBuf) -> Result<Vec<PathBuf>, io::Error>{
+        let mut output: Vec<String> = Vec::new();
+        if dir.is_dir(){
+
+        let  path: Vec<_> = fs::read_dir("dir").unwrap()
```

Comment:
Also, please move this function out of the `impl` block.  It doesn't need to be a method of `Router` but just a normal function defined in this file.